packages/widget-v2/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@skip-go/widget",
   "description": "Swap widget",
-  "version": "3.0.0-alpha.3",
+  "version": "3.0.0",
```

Comment:
looks kinda suspicious

Reply:
hmm, why do previous PRs like this not update widgetv1
https://github.com/skip-mev/skip-go/pull/274/files